hls4ml/templates/vivado_accelerator/arty-a7-100t/tcl_scripts/axi_master_design.tcl

```diff
@@ -57,15 +59,36 @@ apply_bd_automation -rule xilinx.com:bd_rule:microblaze -config { \
 # Enable full FPU
 set_property -dict [list CONFIG.C_USE_FPU {2}] [get_bd_cells microblaze_mcu]
 
+# Create UART interface
+#create_bd_cell -type ip -vlnv xilinx.com:ip:axi_uart16550:2.0 axi_uart
+#apply_bd_automation -rule xilinx.com:bd_rule:axi4 -config { Clk_master {/mig_7series_0/ui_clk (83 MHz)} Clk_slave {Auto} Clk_xbar {Auto} Master {/microblaze_mcu (Periph)} Slave {/axi_uart/S_AXI} intc_ip {New AXI Interconnect} master_apm {0}}  [get_bd_intf_pins axi_uart/S_AXI]
+#apply_bd_automation -rule xilinx.com:bd_rule:board -config { Board_Interface {usb_uart ( USB UART ) } Manual_Source {Auto}}  [get_bd_intf_pins axi_uart/UART]
+
```

Comment:
if not needed, we can remove these commented lines?

Reply:
This may be useful to quickly switch between the UART and UART-lite IPs.